tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -516,3 +534,34 @@ Check And Uninstall Operator In Openshift
         END
     END
     Close All Browsers
+
+Validate URls
+    [Documentation]   validates  urls
+    [Arguments]  ${urls}
+    FOR  ${url}   IN   ${urls}
+        Log To Console  ${url}
+        ${status}=    Check HTTP Status Code    link_to_check=${url}
+        Log To Console     ${url}  gets status code ${status}
+    END
+
+Get Urls and validation of quickStrat Tiles
+    [Documentation]     Clicks on the side window and  validate all Url
+    ${sideWindowButtons}=   Get WebElements   //button[@class='pf-c-wizard__nav-link']
+    FOR    ${sideWindowButton}    IN     @{sideWindowButtons}
+
+            ${status}   Run Keyword And Return Status   Click Element  ${sideWindowButton}
+            IF  ${status} == False
+                Click Button    Next
+            END
+
+            FOR    ${counter}    IN RANGE    5
+                Press Keys    NONE    TAB
+            END
+            Wait Until Element Is Visible  //button[@class='pf-c-wizard__nav-link']
```

Comment:
what is it doing here?

Reply:
waiting  for the Next button to visible

Reply:
sorry I meant to say from line 756 to 758